src/pages/Profile.js

```diff
@@ -24,7 +24,7 @@ const styles = {
   },
   sectionTitle: {
     margin: '2rem 0 1rem',
-    color: colors.darkBlue[0], fontSize: 16, fontWeight: 600, textTransform: 'uppercase'
+    color: colors.accent(), fontSize: 16, fontWeight: 600, textTransform: 'uppercase'
```

Comment:
The header text got bluer:
<img width="311" alt="Screen Shot 2019-05-17 at 8 05 25 AM" src="https://user-images.githubusercontent.com/2395211/57926912-9d2fa780-787a-11e9-8c32-9ab575adb158.png">